spec/src/main/asciidoc/entities.asciidoc

```diff
@@ -18,42 +18,80 @@
 
 == GraphQL Entities
 
+Entities are the objects used in GraphQL. The can be simple objects ("scalars" in GraphQL terminology) or more complex objects
```

Comment:
The should be They (Typo)
They can be simple objects
 (Or maybe It ? - It can be...)

Reply:
Fixed with "They" - good catch!